commit aba8c3753162320c7cc8a66913ad82f4f1fd0d8b
Author: SmallJoker <mk939@ymail.com>
Date:   Thu Jul 30 19:03:48 2020 +0200

    Falling: Fix error caused by missing param2
    
    Falling nodes that were spawned prior the recent falling node changes did not require param2.
    Default to param2 = 0 when none is found in the node data.

diff --git builtin/game/falling.lua builtin/game/falling.lua
index 714506a5f..4bfcca9e7 100644
--- builtin/game/falling.lua
+++ builtin/game/falling.lua
@@ -52,6 +52,7 @@ core.register_entity(":__builtin:falling_node", {
 	floats = false,
 
 	set_node = function(self, node, meta)
+		node.param2 = node.param2 or 0
 		self.node = node
 		meta = meta or {}
 		if type(meta.to_table) == "function" then
